### DISCUSSION TITLE
Photos Moved Instead of Duplicated on Synology NAS

### DISCUSSION THREAD

**Discussion by wulukewu** [TheLastGimbus/GooglePhotosTakeoutHelper#333] [I NEED HELP] [answered]
Hi,

I appreciate the availability of this tool. I have around 3-4 TB of photos generated from Google Photos, which are now stored on my Synology NAS. I tried using tmux and SSH to access the NAS and ran the command like this:

```shell
gpth-linux --input "INPUT_FOLDER_PATH" --output "OUTPUT_FOLDER_PATH" --albums duplicate-copy
``` 

I want the original folder to remain unchanged, so my goal is to duplicate all the photos. However, after processing, I noticed that all the photos and videos in my input folder were moved to the output folder. Did I make a mistake in the command?

Thank you.

**Comment by TheLastGimbus** [comment, marked as answer]
[In readme:](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper?tab=readme-ov-file#4-run-gpth):

> Fun fact: gpth moves files around by default - so if anything goes wrong mid-way, re-export the zips again 🙏

```bash
❯ gpth --help
GooglePhotosTakeoutHelper v3.4.3 - The Dart successor

gpth is ment to help you with exporting your photos from Google Photos.                  
# [ ... ]
    --[no-]copy                   Copy files instead of moving them.
                                  This is usually slower, and uses extra space, but doesn't break your input folder
```

just use `gpth --copy`

**Reply by wulukewu** [reply]
Got it, thanks!
